images/bootstrap/Dockerfile

```diff
@@ -73,23 +104,27 @@ RUN wget -q https://dl.google.com/dl/cloudsdk/channels/rapid/google-cloud-sdk.ta
     gcloud info | tee /workspace/gcloud-info.txt
 
 #
-# BEGIN: DOCKER IN DOCKER SETUP
+# BEGIN: PODMAN IN CONTAINER SETUP
 #
-# Install packages
 
-RUN dnf install -y \
-        kmod \
-        procps-ng \
-        moby-engine && \
-    dnf -y clean all
+RUN dnf -y update; dnf -y reinstall shadow-utils && \
+    dnf -y install crun podman --exclude container-selinux && \
+    dnf -y clean all && \
+    ln -s /usr/bin/podman /usr/bin/docker
```

Comment:
I think that we better auto detect the CRI instead of having the soft link
(kubevirt basically has already auto detect)

Reply:
Ok np

Reply:
I was looking today - the docker command is used quite a bit in the kubevirtci kind providers such as sriov

Reply:
About the last comment please see https://github.com/kubevirt/kubevirtci/pull/826